conductor/src/main/java/com/bluelinelabs/conductor/ControllerFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.bluelinelabs.conductor;
+
+import android.os.Bundle;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public interface ControllerFactory {
+    @Nullable
+    Controller create(@NonNull String controllerName, @Nullable Bundle args);
```

Comment:
Is `controllerClass` or `controllerName` more useful here? e.g. `Controller create(Class controllerClass, Bundle args);`

Reply:
We can't make it Class because of Split APKs.